modules/secretsmanager/main.tf

```diff
@@ -84,14 +84,15 @@ data "aws_iam_policy_document" "orca_kms_key_policy" {
 
     principals {
       type        = "AWS"
-      identifiers = local.iam_users
+      identifiers = local.iam_users 
```

Comment:
Should we make it more open by changing identifier to root instead of orca_users? Not sure if this is also causing the kms issue
```
identifiers = ["arn:aws:iam::${data.aws_caller_identity.current_account.account_id}:root"]
```

Reply:
I believe Nate did this to help with our security posture and limit access to the keys.

Reply:
talked with Nate and decided to open the policy like above for now